ingeniamotion/wizard_tests/absolute_encoder1_test.py

```diff
@@ -0,0 +1,9 @@
+from .feedback_test import Feedbacks
+from .base_test import BaseTest
+from ingeniamotion.enums import SensorType
+
+
+class AbsoluteEncoder1(Feedbacks):
```

Comment:
In my opinion, the classes should have the with test in the name, AbsoluteEncoder1Test, for example. Same for all the classes, including the base one.